{
  "Header": {
    "Original": "CVE-2016-9822 (Integer overflow in libavcodec/mpeg12dec.c in libav 11.8 allows remote ...)",
    "Line": 421753,
    "ID": "CVE-2016-9822",
    "Description": "(Integer overflow in libavcodec/mpeg12dec.c in libav 11.8 allows remote ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-3833-1 DLA-791-1}",
      "Line": 421754,
      "Type": "xref",
      "Bugs": [
        "DSA-3833-1",
        "DLA-791-1"
      ]
    },
    {
      "Original": "- libav \u003cremoved\u003e",
      "Line": 421755,
      "Type": "package",
      "Package": "libav",
      "Kind": "removed"
    },
    {
      "Original": "NOTE: https://blogs.gentoo.org/ago/2016/12/01/libav-multiple-crashes-from-the-undefined-behavior-sanitizer",
      "Line": 421756,
      "Type": "NOTE",
      "Description": "https://blogs.gentoo.org/ago/2016/12/01/libav-multiple-crashes-from-the-undefined-behavior-sanitizer"
    },
    {
      "Original": "NOTE: https://github.com/asarubbo/poc/blob/master/00037-libav-signedintoverflow-mpegvideo_parser",
      "Line": 421757,
      "Type": "NOTE",
      "Description": "https://github.com/asarubbo/poc/blob/master/00037-libav-signedintoverflow-mpegvideo_parser"
    },
    {
      "Original": "NOTE: https://bugzilla.libav.org/show_bug.cgi?id=981",
      "Line": 421758,
      "Type": "NOTE",
      "Description": "https://bugzilla.libav.org/show_bug.cgi?id=981"
    },
    {
      "Original": "NOTE: https://git.libav.org/?p=libav.git;a=commit;h=9f0193c778175cea3fb43f17acf9b90b4d862d33 (pre 11.9)",
      "Line": 421759,
      "Type": "NOTE",
      "Description": "https://git.libav.org/?p=libav.git;a=commit;h=9f0193c778175cea3fb43f17acf9b90b4d862d33 (pre 11.9)"
    },
    {
      "Original": "NOTE: https://git.libav.org/?p=libav.git;a=commit;h=15e1af0006354d6bbf0e433c5d1e8ef13c93d6d0 (pre 11.9)",
      "Line": 421760,
      "Type": "NOTE",
      "Description": "https://git.libav.org/?p=libav.git;a=commit;h=15e1af0006354d6bbf0e433c5d1e8ef13c93d6d0 (pre 11.9)"
    }
  ]
}